README.md

```diff
@@ -222,33 +222,49 @@ Alternative syntaxes for those two cases each have their own flaws; and deciding
 
 As for &lt;language X>, it has different syntactical constraints than JavaScript because of &lt;some construct not supported by X or working differently in X>.
 
-<dt>Ok, but I <b>really</b> think that <i>&lt;alternative syntax></i> is better
+</dd>
+
+<dt>Ok, but I really think that &lt;alternative syntax> is better.</dt>
   
 <dd>
   
 Various alternative syntaxes has been explored and extensively discussed in the past. None of them gained consensus. Search for [issues
 with label “alternative syntax”](https://github.com/tc39/proposal-optional-chaining/issues?utf8=%E2%9C%93&q=label%3A%22alternative+syntax%22), as well as [issues
 with label “alternative syntax and semantics”](https://github.com/tc39/proposal-optional-chaining/issues?utf8=%E2%9C%93&q=label%3A%22alternative+syntax+and+semantics%22) for those that had impact on semantics.
 
-<dt>Why does (null)?.b evaluate to undefined rather than null?
+</dd>
+
+<dt>Why does <code>(null)?.b</code> evaluate to <code>undefined</code> rather than <code>null</code>?</dt>
 
 <dd>
 
 Neither `a.b` nor `a?.b` is intended to preserve arbitrary information on the base object `a`, but only to give information about the property `"b"` of that object. If a property `"b"` is absent from `a`, this is reflected by `a.b === undefined` and `a?.b === undefined`.
 
 In particular, the value `null` is considered to have no properties; therefore, `(null)?.b` is undefined.
 
+</dd>
+
+<dt>Why does <code>foo?.()</code> throw when foo is neither nullish nor callable?</dt>
+
+<dd>
+
+The primary reason is to ensure that `?.` has a consistent semantics in all cases. Instead of making calls a special case where we check `typeof foo === 'function'`, we simply check `foo == null` across the board.
 
+From a usage perspective, one might imagine a library which will call a handler function, e.g. `onChange`, just when the user has provided it. If the user provides the number `3` instead of a function, the library will likely want to throw and inform the user of their mistaken usage. This is exactly what the proposed semantics for `onChange?.()` achieve.
```

Comment:
I might put these two paragraphs in the other order. I don't think we have to identify one of the two as the "primary" reason--they are both valid (and I would weigh the second one higher than the first, personally).

Reply:
Sure, I added a commit that reverses them; hopefully folks can chime in about which they prefer.
Either way, I agree with you that neither reason need be called "primary".